assets/js/components/Root/index.js

```diff
@@ -20,6 +20,11 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
+import {
+	ThemeProvider,
+	createMuiTheme,
+	unstable_createMuiStrictModeTheme, // eslint-disable-line camelcase
```

Comment:
This kind of scary import probably warrants a comment. The git history will often eventually point back here, but let's make it easier for future us 😅

```suggestion
	// We're using the to avoid `StrictMode` warnings. It's a temporary
	// workaround until we can upgrade `material-ui/core` to v5+.
	// See: https://github.com/google/site-kit-wp/issues/5378
	unstable_createMuiStrictModeTheme, // eslint-disable-line camelcase
```